tests/atest/rules/naming/possible-variable-overwriting/expected_output.txt

```diff
@@ -0,0 +1,12 @@
+${rules_dir}${\}test.robot:5:4 [I] 0316 Variable '${MY VAR}' may overwrite similar variable inside 'Test With Overwritten Variables' TestCase
```

Comment:
Do you think it will be clear what's this issue about? Should we either a) mention for which other var it's similar or b) mention how Robot works (ie normalize Robot name)

Reply:
I was thinking about both but:
a) providing variable can be tricky, because sometimes it can be similar to several variables and
b) this makes the message very long but I don't think it's a big issue, so I can add it.